.github/actions/spelling/excludes.txt

```diff
@@ -97,7 +97,7 @@ Resources/(?!en)
 ^doc/reference/UTF8-torture-test\.txt$
 ^doc/reference/windows-terminal-logo\.ans$
 ^NOTICE.md
-^oss/
+^oss/.*?/
```

Comment:
This change allows check-spelling to check `oss/README.md`, which seems reasonable

Reply:
Clever!